iModelJsNodeAddon/JsInterop.cpp

```diff
@@ -922,6 +931,80 @@ DbResult JsInterop::ImportSchemas(DgnDbR dgndb, bvector<Utf8String> const& schem
     return dgndb.SaveChanges();
     }
 
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//---------------------------------------------------------------------------------------
+DbResult JsInterop::ConvertEC2XmlSchemas(bvector<Utf8String> const& ec2XmlStrings, bvector<Utf8String>& ec3XmlStrings, ECSchemaReadContextPtr schemaContext)

```

Comment:
DbResult is not a good return code here. The method is independent of ECDb.

Reply:
The return code has been changed to BentleyStatus type